packages/mini-browser/src/node/mini-browser-endpoint.ts

```diff
@@ -190,7 +190,7 @@ export class MiniBrowserEndpoint implements BackendApplicationContribution, Mini
 }
 
 // See `EditorManager#canHandle`.
```

Comment:
Based on this comment, the `EditorManager#canHandle` value and the new value differ.
https://github.com/theia-ide/theia/blob/78e1cb474e72ed7e641ba7df54ea62b2098684d6/packages/editor/src/browser/editor-manager.ts#L100-L102

Reply:
thank you,  I will change the comment and the variable name